create-publication-from-scopus/src/main/resources/conversion/ignoredScopusFields.txt

```diff
@@ -0,0 +1,4 @@
+.meta
```

Comment:
This file will contain all fields that we know we are ignoring. It has the ability to list nested files as well.

Reply:
Riktig! Ignore Meta. Siden den er Facebook sin.